app/hunter/search_listings.rb

```diff
@@ -1,16 +1,14 @@
 module Hunter
   class SearchListings
-    MAXIMUM_LISTINGS  = 50
+    MAXIMUM_LISTINGS  = 20
     LISTINGS_ENDPOINT = "listings"
 
     DEFAULT_PARAMS = {
                         condition: "used",
-                        item_country: "US",
-                        ships_to: "US",
                         page: "1",
                         per_page: MAXIMUM_LISTINGS,
-                        accepts_gift_cards: true,
```

Comment:
Removing this filter - one of the goals of Hunter is to be able to see the competing listings (listings I would be reselling against) - so limiting my search results to ones that accept DCO prematurely would limit this information. 
Using the listing/show api endpoint I can get the accepted payment methods, so I'll collect & display that data per listing that way.